Chapters/2-8-Interlogue.md

```diff
@@ -0,0 +1,131 @@
+<!--
+
+Interlogue
+
+  - This should be Liffa
+  - Make all of this into the intro of Chapter 15
+
+  - Melna is at Heiko Observatory and sees the craft hurtling into space.
+  - She is furious at Holrag for doing something so reckless, and vows to go back to the senate to demand that they divulge their plans.
+  - She is shown to be handling the heiko issue well while people talk about shaki falling apart in chaos. They suggest that it's safer for her not to go to Shaki.
+  - She considers going directly to Holrag.
+  - She sleeps on it.
+  - Late at night, She gets a knock from a maman named Linyu who demands to speak with her at once. Kipnu sent her.
+
+-->
+
+## Interlogue
+
+"Twelve"
+
+My hands trembled uncontrollably in the darkness of the capsule as it lie still in the launching chamber of the Holragi space rail. I tried checking my shoulder straps and air mask for extra tightness, but my muscles could barely pull anything through the shaking. Fajro assured me that the straps were engineered to hold me tightly against my seat so I wouldn't injure myself during the launch. _I think he cares, but I can't be sure._
+
+"Eleven"
+
+The tumblers from all the gauges burned their zeroes into my eyes. I went over what I had been taught. _The altimeter is on the left, measured in paces. Stable trajectory at 8000 paces, but I'll have to use the thruster knobs to control my orientation. Orientation! That's over-_
+
+"Ten"
+
+_Dammit it's too fast! Orientation is to the right of the altimeter._ I spun the ball a bit to get a feel for how I might use it once I got up there. _See, scribe, spin. Once every Va'la. Three red dots opposite Ara. One big red dot for Ara._
+
+"Nine"
+
+The voice of a control officer filled my listener, "Capacitance: stable. Temperature: stable."
+
+I instinctively responded, "Thank you," with my mask still on. But no one could hear me.
+
+"Eight"
+
+My whole body tensed up again. The anxiety came in waves with the countdown. I clenched my right hand around the steel knife-bracelet they had given me in the case of my death. _There's a reason I have this. I might die._
+
+_"It's only dangerous if you deviate from your mission."_ Jathi's reassurance calmed me, but in the back of my mind I knew that none of them could be certain.
+
+"Seven"
+
+"Ahhhnnnhhhh!" I screamed out to fight the panic, and seethed through my teeth at the pain my whole body and mind were in. _Breathe. Breathe. I need to focus to stay alive._
```

Comment:
> I screamed out to fight the panic, and seethed through my teeth at the pain my whole body and mind were in.

Yes, that's probably why people scream in rollercoasters and scary movies. But if one has the time to be aware that is why they are screaming, they won't be screaming. It's unrealistic of Liffa to be this aware while screaming.